ts/src/bitmart.ts

```diff
@@ -2319,13 +2336,35 @@ export default class bitmart extends Exchange {
         if (postOnly) {
             request['mode'] = 4;
         }
+        const triggerPrice = this.safeStringN (params, [ 'triggerPrice', 'stopPrice', 'trigger_price' ]);
+        const isTriggerOrder = triggerPrice !== undefined;
+        const trailingTriggerPrice = this.safeString2 (params, 'trailingTriggerPrice', 'activation_price', price);
+        const trailingStopPercent = this.safeString2 (params, 'trailingStopPercent', 'callback_rate');
```

Comment:
@Dan-krm should we use `trailingPercent` instead?

Reply:
Yeah sure I'll make the changes